starknet-foundry/crates/forge/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "forge"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Imo we should do a patch instead 0.1.1

Reply:
We are changing the behavior of test collecting, it's not just a fix